lesson-27/src/components/Layout.jsx

```diff
@@ -0,0 +1,12 @@
+import { Outlet } from 'react-router'
+import Header from './Header/Header.jsx'
+
+export default function Layout() {
+  return (
+    <>
+      <Header />
+      <main><Outlet /></main>
+      <footer />
```

Comment:
```suggestion
      <main>
        <Outlet />
      </main>
```

чуть заметнее вложенность, футер ничего не делает - для него нет никаких стилей